README.md

```diff
@@ -200,11 +266,143 @@ df <- read.df(table_path, "deltaSharing")
 
 You can try this by running our [examples](examples/README.md) with the open, example Delta Sharing Server.
 
+### CDF 
+Starting from release 0.5.0, querying [Change Data Feed](https://docs.databricks.com/delta/delta-change-data-feed.html) is supported with delta sharing.
+The provider can turn on the CDF on the original delta table, and share it in delta sharing. Once CDF is shared, the recipient can query
+CDF of a delta sharing table in a similar way of a delta table:
+```scala
+val tablePath = "<profile-file-path>#<share-name>.<schema-name>.<table-name>"
+val df = spark.read.format("deltaSharing")
+  .option("readChangeFeed", "true")
+  .option("startingVersion", "3")
+  .load(tablePath)
+```
+
+### Streaming
+Starting from release 0.6.0, delta sharing table can be used as the data source for [spark structured streaming](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html).
+The provider can share the table with history, then the recipient can streaming query the table:
```

Comment:
Consider rephrasing:
Once the provider shares a table with history, the recipient can perform a streaming query on the table.

Reply:
updated.